### ISSUE TITLE
Steam Broken | Minimize to tray missing | Feature requests

### ISSUE THREAD

**Issue by rnbdsh** [imprecision/franz-app#10]
Arch linux x64, version 0.9.10-3 (newest atm) from yaourt

#1: After logging into steam the specified tab crashes. Switching user language from DE to EN did not work (when logged in browser)
#2: Currently i see no signs of the announced minimize to tray feature even though the changelog says it*s in my version

Feature requests:
#3: The program could automatically check the "remember me" options (probably there is an url-param for that or via JS injection?)
#4: A feature for notfying me when i get a new google mail would be nice. Not directly a messenger but kind of.
#5: A tiled version with 2/4 on one screen would be nice, or all sidebars in one view next to each other, clicking on one opens the according chat window in the main window. (I know this is hard, and DOM manipulation)

Greetings from Linz/Austria.

**Comment by imprecision** [comment]
Hi, sorry to hear you're having troubles. The crashing tab - have you tried deleting the service and then re-adding it? If this doesn't help - could you let me know what DE you're running? We'll then need to spend some time testing similar envs here.

Minimise to tray - that depends on the DE I'm afraid, so if you let me know which you're on I'll add it to our test list.

Everything else - great ideas! :) A few of them we have related plans on so future versions soon we hope should get much better for you. 

Cheers,
-Jonathan

**Comment by rnbdsh** [comment]
I use cinnamon and i just also tried gnome - did not work with those two. I already got a tray icon for franz, but it's also in the task bar and when i close it, it isn't hidden, Also hide/show in taskbar does not work.
Removing and adding the steam tab again did not change anything.
Nice.

**Comment by imprecision** [comment]
Right, ok, thanks for the info - I need to go away and do some testing!
